itsnameissteven/metAPI
# Feature/api detail page

## What does this PR do?
1. Detail page has been built out 
2. notes can be saved for a specific API
3. API can be favorited
4. Data persists via localStorage
-

## How should it be tested in the UI?
click on an api card to go to the detail page 
write and save a note, the note should render on the right of page.
save an api as a favorite. this will not show up anywhere but can be found in local storage

-
